www/js/diary/current.js

```diff
@@ -84,15 +84,20 @@
       sel_region: null
   };
 
-  var startTimeFn = function(ts) {
-    var date = new Date(ts*1000);
-    var hours = date.getHours();
-    var minutes = date.getMinutes();
-    var amOrPm = hours < 12 ? 'AM' : 'PM';
-    hours = hours % 12;
-    hours = hours ? hours : 12;
-    minutes = minutes < 10 ? '0'+ minutes : minutes;
-    return hours + ':' + minutes + ' ' + amOrPm;
+  var startTimeFn = function (ts) {
+    var date = new Date(ts * 1000);
+    if ($translate.use() != "en") {
+      return moment(date).format('HH:mm');
+    } else {
+      return moment(date).format('HH:mmA')
```

Comment:
Thanks for doing this in the principled way. But I'm curious about why you use the am/pm format only when the current language is in english. Does it not work for French? Have you tried the locale aware formats (e.g. `LT`). I will not hold up the merge for this, but you might want to test and fix in a second round.

Reply:
AM/PM does not work in French as most of the french speaking people think that AM meaning is "après-midi" (afternoon). But the good news is that locale aware formats seem to work ! So I will trust `moment` now (https://github.com/e-mission/e-mission-phone/pull/584/commits/f8db039d97f3d403f7a1dbd16174c5f0c4423fee). Thank you for your suggestion.